package.json

```diff
@@ -22,6 +22,7 @@
     "test": "ember try:each"
   },
   "dependencies": {
+    "broccoli-concat": "^3.2.2",
```

Comment:
sorry, I think you misunderstood me in the issue.  You do not need to pin the version here like this, but you need to update your `yarn.lock` so that you can get the newer version. Simplest solution is to revert this change, and do `rm yarn.lock && yarn install` to update to the latest of all transitive deps.